spec/models/recipe_revision_spec.rb

```diff
@@ -1,4 +1,16 @@
 require 'spec_helper'
 
 describe RecipeRevision do
+  let(:revision) { FactoryGirl.create(:recipe_revision, revision: 1, recipe_id: 1) }
```

Comment:
I think these should be created in the specs, multiple levels of nesting lets ends in very confusing specs. Similar to the specs you added to `spec/models/recipe_spec.rb`